src/app/components/normal-component/lobby/lobby.component.spec.ts

```diff
@@ -324,18 +324,6 @@ describe('LobbyComponent', () => {
         // Then it should have unsubscribed from active parts
         expectUnsubscribeToHaveBeenCalled();
     }));
-    it('should unsubscribe from active users when destroying component', fakeAsync(async() => {
-        // Given an initialized lobby
-        const expectUnsubscribeToHaveBeenCalled: () => void =
-            prepareUnsubscribeCheck(TestBed.inject(ActiveUsersService), 'subscribeToActiveUsers');
-        testUtils.detectChanges();
-
-        // When it is destroyed
-        component.ngOnDestroy();
-
-        // Then it should have unsubscrbed from active users
-        expectUnsubscribeToHaveBeenCalled();
-    }));
```

Comment:
why does it suddently need to die ?

Reply:
j'avais mal résolu les conflits en pullant develop, les active users n'existent plus en develop, donc ce test est mort !